src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -316,9 +316,11 @@ public BlockGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 				L10N.t("elementgui.block.event_on_redstone_off"),
 				Dependency.fromString("x:number/y:number/z:number/world:world/blockstate:blockstate"));
 		onHitByProjectile = new ProcedureSelector(this.withEntry("block/on_hit_by_projectile"), mcreator,
-				L10N.t("elementgui.common.event_on_block_hit_by_projectile"),
-				Dependency.fromString(
-						"x:number/y:number/z:number/world:world/entity:entity/direction:direction/blockstate:blockstate/hitX:number/hitY:number/hitZ:number"));
+				L10N.t("elementgui.common.event_on_block_hit_by_projectile"), Dependency.fromString(
+				"x:number/y:number/z:number/world:world/entity:entity/direction:direction/blockstate:blockstate/hitX:number/hitY:number/hitZ:number"));

```

Comment:
Please revert the changes on this line, as it's unrelated to the PR

Reply:
Done!